package/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all
+all:
+	docker build -t virtualenv-tools-fpm .
+	docker run \
+	  --env PACKAGE_ITERATION="$(PACKAGE_ITERATION)" \
+	  --env PACKAGE_MAINTAINER="$(PACKAGE_MAINTAINER)" \
+		--env APP_VERSION="$(APP_VERSION)" \
+	  --tty \
+	  --rm \
+	  --volume $(PWD)/..:/build \
```

Comment:
Because `fpm` needs to access the virtualenv code, so that has to be visible in the docker container. Because we run make in the `package` directory, we have mount the upper directory, but the `package` script moves the file to the `package` directory at the end, so the `deb` file will appear where you expect it :)